README.md

```diff
@@ -64,40 +64,68 @@ Installation Requirements:
     $ make integration_test
     ```
 
-## Deployment
+## AWS Deployment
 
-In addition to the steps above, and assuming the application is located on: /home/ubuntu/genomic-data-service/:
+A production grade data services deployment consists of three machines:
+* Main machine that runs the flask app that sends the requests to the ES machines.
+* Regulome search ES
+* ENCODED region-search ES
 
-1. Install the following packages:
+1. Deploy the machines. Make sure you have activated the virtual environment created above.:
     ```
-    $ sudo apt-get update
-    $ sudo apt-get install python3-psycopg2 redis-server apache2-utils nginx
+    $python deploy/deploy.py -b <your-branch-here> -n test-data-service  --image-id ami-0787b9e5747b9e78c  --profile-name regulome --instance-type r5.2xlarge --volume-size 500
+    python deploy/deploy.py -b <your-branch-here> -n test-data-service-regulome  --image-id ami-0787b9e5747b9e78c  --profile-name regulome --instance-type r5.2xlarge --volume-size 500
+    python deploy/deploy.py -b DSERVE-77-aws-deploy -n test-data-service-encoded  --image-id ami-0787b9e5747b9e78c  --profile-name regulome --instance-type r5.2xlarge --volume-size 500
```

Comment:
missing '$' i the front. Should change "DSERVE-77-aws-deploy" to "<your-branch-here>"